alf/algorithms/oac_algorithm.py

```diff
@@ -109,14 +73,15 @@ def __init__(self,
         Refer to SacAlgorithm for Args besides the following.
 
         Args:
-            explore (bool): whether to use the explore policy in rollout_step. Only
-                continuous action space is supported when `explore` is True.
+            explore (bool): default is True for OAC algorithm, where 
+                'unroll_with_grad' has to be True in the 'TrainerConfig' 
```

Comment:
This requirement is not necessary. You can use the following to enable gradient:
```python
with torch.enable_grad():
   ...
```

Reply:
updated.